config/prow/labels.yaml

```
@@ -1678,6 +1678,8 @@ repos:
     labels: *trigger-only-labels
   gardener/gardener-extension-shoot-rsyslog-relp:
     labels: *default-labels
+  gardener/gardener-extension-shoot-cert-service:
+    labels: *trigger-only-labels
```

```suggestion
    labels: *default-labels
```
Since the PR enables `tide` too.